endpoints/events/event_test.go

```diff
@@ -424,6 +435,26 @@ func TestHandleAccountServiceErrors(t *testing.T) {
 				response: "Invalid request: Prebid-server has disabled Account ID: testacc, please reach out to the prebid server host.\n",
 			},
 		},
+		"timeout": {
+			fetcher: &mockAccountsFetcher{
+				Fail:       false,
+				DurationMS: 50,
```

Comment:
Does this imply the test runs for 50ms? Is there way to perhaps complete the context immediately to simulate the timeout? Just trying to keep "sleeps" during tests minimal.

Reply:
Not quite, the mocked _FetchAccount_ relies on a select with two communications and blocks until one of them completes.
DurationMS indicates how long one of the two operations will take to complete, the other is controlled by the _context_ received as an argument.
The _context_ is configured with a timeout set in the configuration, which, in this case is set to **1ms**.
In essence, the _FetchAccount_ will unblock after 1ms due to our context timing out and complete soon after.